src/lotus_json/signature_type.rs

```diff
@@ -4,11 +4,16 @@
 use super::*;
 use crate::shim::crypto::SignatureType;
 
+// Lotus uses signature types under two names: `KeyType` and `SigType`.
+// `KeyType` can be deserialized from a string but `SigType` must always be an
+// integer. For more information, see
```

Comment:
Shouldn't we add lenses for `SignatureType` to cover these two cases?

Reply:
Not sure what you mean by this. Matching Lotus' behavior exactly will require some refactoring that is out of scope of this PR.